Documentation/supported-mod-list.md

```diff
@@ -39,7 +39,14 @@
 # List of Half-Life-based mods
 Originally this list was written by *Qwertyus*(*Qortez*)
 
-To run mods from this category - place mod folder to folder where is **valve** folder located and launch xash3d fwgs with command-line parameter `-game <modfolder>`.
+To install these mods:
+- place mod folder in the same directory as your **valve** folder is located
```

Comment:
Capitalize the first letter on the list.

Plus, it could be turned from bulleted to numbered list.

Reply:
An old instruction was universal for all platforms, but new instruction looks more for PC.
`Custom game` menu is not always available anyway.
I think "To install and run these mods" will be more correct here.